site/content/pages/ladds-500.md

```diff
@@ -28,5 +28,5 @@ The Ladds 500 is annual event. It takes place each April, the day after that mon
 
 ## Support
 
-Help make The Ladds 500 happen! [Donate to support The Ladds 500.](https://www.paypal.com/donate/?cmd=_s-xclick&hosted_button_id=JB8YUKRB2KPV4)
+Help make The Ladds 500 happen! [Donate to support The Ladds 500.](https://www.paypal.com/donate/?cmd=_s-xclick&hosted_button_id=JB8YUKRB2KPV4 target="_blank")
```

Comment:
Markdown apparently doesn't like attributes in the link (renders this text literally on the page). We can change to HTML though, since Markdown content support inline HTML anywhere.